packages/kbn-performance-testing-dataset-extractor/src/cli.ts

```diff
@@ -58,8 +58,9 @@ export async function runExtractor() {
       const scalabilitySetup: ScalabilitySetup = config.get('scalabilitySetup');
 
       if (!scalabilitySetup) {
-        log.error(`'scalabilitySetup' must be defined in config file!`);
-        return;
+        log.warning(
+          `'scalabilitySetup' is not defined in config file, output file for Kibana scalability run won't be generated`
+        );
```

Comment:
I assume you didn't mean to remove the `return` here

Reply:
I did mean it. Today I discussed with @danielmitterdorfer that extractor produces json output for APM data only if journey FTR config has 'scalabilitySetup' defined and it is opt-in option.
From Daniel perspective we should ship ES queries for each single performance journey not depending on FTR config because it is targeted for Elasticsearch engineers, who might not be familiar with the config and its recent changes. I think it is good point. 
I remove return so that we don't exit the script, added warning and a check to skip saving Kibana requests output file if scalabilitySetup is not defined.

Reply:
Sounds good to me